src/forms/FormFieldMapCoordinates/MapCoordinatesDrawer/MapCoordinatesDrawer.tsx

```diff
@@ -187,7 +190,15 @@ const MapCoordinatesDrawer = (props: MapCoordinatesDrawerProps): ReactElement =>
 				Click on the map to update the lattitude and longitude coordinates
 			</StyledSpan>
 		</>
-	);
+	)
+
+	const onBlurLatitude = (latValue: number) => {

```

Comment:
I couldn't manage to make this work because since the callbacks were passed as properties to the fields and the fields array gets memoized, then the functions were never able to get the newest mapPosition, and when trying to add the functions to the dependency array of fields (either using a useCallback or useMemo) the entire drawer form would get re-rendered over and over.

I ended up trying another approach like this
![image](https://user-images.githubusercontent.com/23409284/185523068-a9228a5a-0c99-47e9-96be-026e070c1cb8.png)

Let's talk about it tomorrow morning and if you agree I'll push directly from what I have.